linting/stylelint-config-bitcrowd/package.json

```diff
@@ -28,14 +28,14 @@
   "homepage": "https://github.com/bitcrowd/frontend#readme",
   "dependencies": {
     "object-assign": "^4.1.0",
-    "stylelint-config-standard": "^17.0.0",
-    "stylelint-order": "^0.6.0"
+    "stylelint-config-standard": "^19.0.0",
+    "stylelint-order": "^3.1.1"
   },
   "peerDependencies": {
     "stylelint": "^8.1.1"
   },
   "devDependencies": {
-    "eslint": "^3.9.1",
+    "eslint": "^6.6.0",
```

Comment:
I wonder why this is specifying ESLint as a development dependency here. Should we configure CI for this too?

Reply:
It needs eslint, because the `test` is just linting all the files against our own eslint config.